source/Calamari.Aws/Integration/AwsAuthConvention.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using Calamari.CloudAccounts;
+using Calamari.Common.Commands;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment.Conventions;
+
+namespace Calamari.Aws.Integration
+{
+    public class AwsAuthConvention : IInstallConvention
```

Comment:
I went back to use the `AwsAuthConvention` because I was getting issues in tests etc when I tried to add it to `SetupKubectlAuthentication`. We can combine it later I guess but for now it's easier to just keep it as a separate convention.

Reply:
CC @zentron